python/server/ServerWrapper.cpp

```diff
@@ -116,7 +119,7 @@ namespace {
         } else {
             Empire* empire = GetEmpire(empire_id);
             if (!empire) {
-                ErrorLogger() << "PythonServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
+                ErrorLogger() << "ServerWrapper::GenerateSitRep: couldn't get empire with ID " << empire_id;
```

Comment:
Hm, question is, is there a C++ feature that lets you reference the filename of the source file? If yes, that could be used, if not, I don't see a way to achieve this, however complicated.